### ISSUE TITLE
Considder upgrading to latest liip-imaginebundle

### ISSUE THREAD

**Issue by tarjei** [Kunstmaan/KunstmaanBundlesCMS#1457]
So we get more options for optimizing images :)

I tried with: 
```
        "liip/imagine-bundle": "1.7 as 1.4.3",
        "imagine/imagine": "0.6.3 as 0.5.0",
```

And would love to hear if you have had any issues that can affect this setup.

Regards,
Tarjei

**Comment by aistis-** [comment]
We also used manual updating of `liip/imagine-bundle` :thinking:  would nice to have this update in KunstmaanCMS too.

**Comment by Devolicious** [comment]
Hi guys,

Atm we cannot yet do this. As I already said in #1459, we need to investigate it before we can update. When we have more info on this we will come back on this issue.

**Comment by tarjei** [comment]
Hi, what would be nice is if it was possible to choose. I can understand that you might need to lock this for some internal needs but couldn't those needs be satisfied with a custom filter ?
